test/distributed-ranges/shp/CMakeLists.txt

```diff
@@ -19,18 +19,19 @@ else ()
     message(STATUS "suitable for distributed ranges cmake version ${CMAKE_VERSION} found")
 endif()
 
+add_library(dr_shp INTERFACE)
+add_library(DR::shp ALIAS dr_shp)
```

Comment:
can we implement it without this two variables dr_shp and DR::shp. This is something which is either empty or having `-DUSE_FMT`?